src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+- name: Checking DNS resolution first
+  shell: |
+    dig ipa-ca.{{ ipa_domain }} | grep NOERROR
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 10
+
+- name: Make sure virtual smartcard dir exists
+  file:
+    path: "{{ virt_smartcard_dir }}"
+    state: directory
+
+- block:
+  - name: Run IPA advise script for server setup for smart cards
+    shell: |
+      kinit admin@{{ ipa_domain.upper() }}
+      ipa-advise config-server-for-smart-card-auth > {{ virt_smartcard_dir }}/sc_server.sh
+      sh -x {{ virt_smartcard_dir }}/sc_server.sh {{ sssd_ca_path }} {{ ipa_ca_path }}
+    args:
+      stdin: '{{ ipa_password }}'
+    register: ipa_advise_sc_server
+
+  - name: Workaround to disable IPA WebUI OCSP checking
+    lineinfile:
+      path: /etc/httpd/conf.d/ssl.conf
+      regexp: 'SSLOCSPEnable on'
+      line: 'SSLOCSPEnable on no_ocsp_for_cert_ok'
```

Comment:
Regexp match the line so if you run this two times it might not work correctly.

`regexp: SSLOCSPEnable on\\s*$` 

might be better.

Reply:
Ah good point.   I'll adjust the regexp.